G/GR/build_tarballs.jl

```diff
@@ -90,15 +90,15 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency("Bzip2_jll"),
-    Dependency("Cairo_jll"),
+    Dependency("Bzip2_jll"; compat="1.0.8"),
+    Dependency("Cairo_jll"; compat="1.16.1"),
```

Comment:
When i install just Qt6Base_jll it pulls in `Cairo_jll v1.18.4`, so possibly this needs to be bumped too.